brokers/vscode/cmd/manual-testing-several-extensions-in-one-plugin.json

```diff
@@ -1,41 +0,0 @@
-[
-  {
-    "id": "org.eclipse.che.vscode-redhat.java-and-yaml-in-sidecar",
```

Comment:
Name of this file gives pretty clear what is expected during manual testing. I see that now `config.json` for unified broker contains all plugins with all plugins types and it is not clear what to expect as a result of work. It may be an issue to check result of unified broker work if there are so much to do, like check that sidecar is generated for Theia Remote Plugins, multiple vscode extensions are run in the same sidecar. Maybe it would be better to split `config.json` into separated files?
BTW It is not really related to this PR but just proposal for further improvement of unified Broker.

Reply:
Yeah, I was thinking about writing some integration tests. Some case are trickier to test though.